samples/spring-client-java-valueentity-counter/src/main/proto/com/example/counter_api.proto

```diff
@@ -0,0 +1,47 @@
+// This is the public API offered by your entity.
+syntax = "proto3";
+
+package com.example; // <1>
+
+import "akkaserverless/annotations.proto"; // <2>
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+option java_outer_classname = "CounterApi"; // <3>
+
+message IncreaseValue { // <4>
+  string counter_id = 1 [(akkaserverless.field).entity_key = true]; // <5>
```

Comment:
same as above
the annotation on `counter_id` should be remove and all the other examples here below

Reply:
Removed akkaserverless annotations from all the proto files.